### DISCUSSION TITLE
Remove the dimension of time from activities

### DISCUSSION THREAD

**Discussion by zanebclark** [openmobilityfoundation/curb-data-specification#94] [Data Specification]
This discussion is a spin out from https://github.com/openmobilityfoundation/curb-data-specification/discussions/64

The current list of [activities](https://github.com/openmobilityfoundation/curb-data-specification/tree/main/curbs#activities) conflates two separate dimensions: time and activity. For example, `parking` and `stopping` are only distinguishable by time. This poses a few problems:
1. Activities now have a quasi-hierarchy where one activity implies that another activity is allowed. For example, `parking` also allows `stopping`. As additional activities are added, informing and enforcing this hierarchy will become more complicated.
2. It's possible to define a Rule with a `max_stay` that is lower than the duration of the allowed activity (e.g. `parking`).
3. What is the allowed duration for `parking` or `stopping`? Where is that defined? Is it `user_class` dependent?

I propose that activities with a time-based definition be removed from the list of activities (`parking`, `stopping`). To reflect these time-based activities, `max_stay` should be used instead. In response to the points above: 
1. Quasi-hierarchy: No more hierarchy to enforce or inform
2. max_stay less than activity duration: There's no longer an overlap of responsibility here
3. Activity duration definition: Activities no longer require a duration definition. `max_stay` can be used to make the duration of the activity `user_class`-dependent.

**Comment by jlarsonOmahaNE** [comment]
@zanebclark Do you think that the activity descriptions just need to be refactored?

`parking` - may park and leave vehicle unattended
`no parking` - may not park and leave vehicle unattended
`loading` - loading of goods 
`no loading` - no loading of goods
`unloading` - unloading of goods
`no unloading` - no unloading of goods
`stopping` - stopping briefly to pick up or drop off passengers
`no stopping` - stopping, loading, unloading, and parking are all prohibited; not a typical travel lane
`travel` - represents curbside lanes typically intended for moving vehicles, like bus lanes, bike lanes, and rush-hour-only travel lanes; implies that parking, loading, unloading, and stopping are prohibited.
`no travel` - no travel is allowed, implies it could be used for other activities
